bin/glpi-agent

```diff
@@ -258,6 +259,9 @@ glpi-agent [options] [--server server|--local path]
     --backend-collect-timeout=TIME timeout for inventory modules
                                      execution (30)
     --additional-content=FILE      additional inventory content file
+    --assetname-support=1|2        if 1 (the default), the short hostname is used as asset name
+                                   if 2, the as-is hostname (can be fqdn) is used as asset name
+				   Warning: this feature does not apply on MacOS or Windows
```

Comment:
The same as in Changes file, a warning is wrong here:
```suggestion
    --assetname-support=1|2        [unix/linux only] set the asset name depending on the given value:
                                    - 1 (the default), the short hostname is used as asset name
                                    - 2, the as-is hostname (can be fqdn) is used as asset name
				                   this feature is not supported on MacOS or Windows
```

Reply:
Just noticed also there were few tabulations on the last line.

The same should be applied too fro the glpi-inventory script.